### ISSUE TITLE
Bug: Enabling sidekick breaks the VIM mode

### ISSUE THREAD

**Issue by HEmile** [hadynz/obsidian-sidekick#29]
See video:

https://user-images.githubusercontent.com/1289165/155547715-d310cb85-83c3-4740-963e-ba6dcab827f3.mov

I turned Sidekick off, and then this behavior stopped. What happens is, when entering _insert mode_ in VIM, at random points it starts interpreting inputs as if outside insert mode even though I never click on escape. 
Someone else reported the same problem on Discord.

**Comment by shabegom** [comment]
Yep, saw this issue as well. It randomly switches to Normal mode as if I had hit escape.

**Comment by HEmile** [comment]
@hadynz This bug makes the plugin unusable for me. Is there a chance this can be resolved soon? Otherwise I'll give it a shot myself.

**Comment by HEmile** [comment]
Weirdly enough I cannot reproduce this anymore. Maybe a downstream bug fix from Obsidian? @shabegom do you still experience this?
